features/steps/305.Big_Data_Spark.py

```diff
@@ -0,0 +1,66 @@
+import os.path
+
+from behave import given, when, step, then
+from config.settings import CODE_HOME
+import os, stat
+from os.path import join, isdir, exists, basename
+from os import chdir, getcwd, environ, listdir
+from requests import get, post
+from requests import ConnectionError, HTTPError, URLRequired, Timeout, TooManyRedirects
+from logging import getLogger
+from features.funtions import check_java_version
+import subprocess
+from re import match
+from warnings import warn
+from features.timeout import Timeout
+from datetime import datetime, timedelta
+from pytz import UTC, timezone
+from os import environ
+import time
+
+__logger__ = getLogger(__name__)
+
+new_file = ''
+jar_file_id = ''
+subscription_id = ''
+
+
+@given(u'I set the tutorial 305.Spark')
+def step_impl(context):
+    context.data_home = join(join(join(CODE_HOME, "features"), "data"), "305.Big_Data_Spark")
+
+
+@given(u'I prepare the script {script_name} to be run')
+def step_impl(context, script_name):
+    filename = join(context.data_home, script_name)
+    assert(os.path.exists(filename))
+    os.chmod(filename, stat.S_IRWXU | stat.S_IRWXG)
+    context.script_name = filename
+
+
+@step(u'I set the environ variable {var_name} to "{var_value}"')
+def step_impl(context, var_name, var_value):
+    environ[var_name] = var_value
+
+
+@step(u'I set the environ variable {var_name} to "{var_value}" under git')
+def step_impl(context, var_name, var_value):
+    environ[var_name] = join(context.parameters['git-directory'], var_value)
+
+
+@when(u'I exec the script as in the tutorial page')
```

Comment:
@when(u'I execute the script as in the tutorial page')

Reply:
I run the script.... better, I think

Reply:
ok

NTC